wallet/src/Cardano/Wallet/API/Internal.hs

```diff
@@ -3,13 +3,15 @@
 -- Daedalus client, and aren't useful for wallets, exchanges, and other users.
 module Cardano.Wallet.API.Internal where
 
+import Prelude
```

Comment:
Indentation to match import lines below?  Also isn't this import already implicit?